notifications_utils/timezones.py

```diff
@@ -1,32 +1,44 @@
-from datetime import datetime
+import os
 
 import pytz
 from dateutil import parser
 
-local_timezone = pytz.timezone("Europe/London")
+local_timezone = pytz.timezone(os.getenv("TIMEZONE", "America/New_York"))
 
 
 def utc_string_to_aware_gmt_datetime(date):
     """
-    Date can either be a string, naive UTC datetime or an aware UTC datetime
-    Returns an aware London datetime, essentially the time you'd see on your clock
+    Date can either be a string, naïve UTC datetime or an aware UTC datetime
+    Returns an aware local datetime, essentially the time you'd see on your clock
     """
-    if not isinstance(date, datetime):
-        date = parser.parse(date)
-
+    date = parser.parse(date)
```

Comment:
Does `parser.parse` work if `date` is already a datetime? I'm worried about removing that guard since this library is pulled in from more than just the api repo

Reply:
Nope. I think this is also a candidate for deletion, though, so let me try that first.